### PR TITLE
Add UI to display five star and average rating in Skill Details page (fixed #1324)

### LINKED ISSUE #1270: Display the 5-star ratings on the Skills page
**Actual Behaviour**
Currently, thumbs-up and thumbs-down rating is shown.

**Expected Behaviour**
It should display it according to this image - 
screenshot from 2018-05-22 00-24-54

Ref - https://developers.google.com/actions/distribute/directory

**Steps to reproduce it**
Open the skills page

**Would you like to work on the issue?**
Anyone is free to work on it.

### LINKED ISSUE #1324: Add UI for five star and average rating system in skill details page
**Actual Behaviour**

Currently, there is a thumbs up/down rating implemented in both the chat activity as well as the skill details screen. 

**Expected Behaviour**

Add UI for implementing the new five star and average rating system.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by arundhati24**
Fixes #1324  [Parent issue #1270 ]

Changes: Add Skill Ratings chart in place of existing rating in SkillDetailsFragment.

screenshot_1527708775

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Always add screenshots for UI changes

**Comment by arundhati24** [comment]
@iamareebjamal Just added a screenshot.

File: app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/MyXAxisValueFormatter.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package org.fossasia.susi.ai.skills.skilldetails;
+
+import com.github.mikephil.charting.components.AxisBase;
+import com.github.mikephil.charting.formatter.IAxisValueFormatter;
+
+public class MyXAxisValueFormatter implements IAxisValueFormatter {
+
+    private String[] mValues;
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/MyXAxisValueFormatter.java]: Don't use hungarian notation

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -57,6 +57,7 @@
     <dimen name="standard_content_padding">5dp</dimen>
 
     <dimen name="text_size_large">20sp</dimen>
+    <dimen name="text_size_average_rating">56sp</dimen>
```
Comment (iamareebjamal) [inline app/src/main/res/values/dimens.xml]: Use text_size_very_large

**Comment by iamareebjamal** [comment]
@arundhati24 Use material colors and pick lighter colors. Particularly blue https://www.materialui.co/colors

**Comment by arundhati24** [comment]
@iamareebjamal I changed the colors. Please review.

File: app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt [hunk 3]
```diff
@@ -32,6 +43,10 @@ class SkillDetailsFragment : Fragment() {
     lateinit var skillTag: String
     val imageLink = "https://raw.githubusercontent.com/fossasia/susi_skill_data/master/models/general/"
 
+    lateinit var fiveStarSkillRatingBar : RatingBar
+    lateinit var fiveStarSkillRatingScaleTextView : TextView
+    lateinit var skillRatingChart: HorizontalBarChart
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt]: These can be private

**Pull request merged**